src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -175,6 +175,11 @@ let workbenchProperties: { [path: string]: IJSONSchema; } = {
 		'type': 'boolean',
 		'description': nls.localize('closeOnFileDelete', "Controls if editors showing a file should close automatically when the file is deleted or renamed by some other process. Disabling this will keep the editor open as dirty on such an event. Note that deleting from within the application will always close the editor and that dirty files will never close to preserve your data."),
 		'default': true
+	},
+	'workbench.fontAliasing': {
+		'type': 'boolean',
```

Comment:
Would it not make more sense to expose three options as enumeration here? `none`, `subpixel-antialiased` and `antialiased`? I find it a bit weird to say `workbench.fontAliasing: true` because this implies that by default there is no font smoothing happening, which is not true. The default in Chrome seems to be `subpixel-antialiased`